components/cam/bld/configure

```diff
@@ -2658,6 +2658,7 @@ sub write_filepath
       }
       elsif($opts{'dyn_target'} eq 'theta-l') {
         print $fh "$camsrcdir/homme/src/theta-l\n";
+        print $fh "$camsrcdir/homme/src/theta-l/share\n";
```

Comment:
I'd suggest keeping this PR homme-only, no cam changes (which should be the next PR).

Reply:
well, without that change, cam won't find some files. In order for theta-l_kokkos to reuse some f90 files from theta-l, I did what was done for preqx, and created the share folder (I think we discussed it on slack, and that was the agreed upon strategy). So some files have been moved from theta-l to theta-l/share.

Reply:
Is it needed to keep CAM target theta-l functional? if not, let's keep it out of this PR. btw I will run cam tests anyway before this is merged.